docs/conf.py

```diff
@@ -20,12 +21,13 @@
 # -- Project information -----------------------------------------------------
 
 project = "Bayesian Tracker (btrack) 🔬💻"
-copyright = "2022, Alan R Lowe"
 author = "Alan R Lowe"
+copyright = f"2022-{datetime.date.today().year}, {author}"
 
 
 # The full version, including alpha/beta/rc tags
-release = "v" + btrack.__version__
+version = f"v{btrack.__version__}"
+release = version
```

Comment:
is `version` needed here? could you do:
```python
release = f"v{btrack.__version__}"
```

Reply:
https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information

> If you don’t need the separation provided between [version](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-version) and [release](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-release), just set them both to the same value.

I took this to mean you should have both

Reply:
yeah that does sounds like both are needed